### DISCUSSION TITLE
Visual Testing

### DISCUSSION THREAD

**Discussion by shanmswa** [webdriverio/webdriverio#12877] [Community]
I am using the browser.saveElement() method to save an element. I am able to save the element but without its text. 
<img width="137" alt="Actual button" src="https://github.com/webdriverio/webdriverio/assets/113699426/3eea47a3-03d7-4219-9e0e-cb21e2f6215c">
![image](https://github.com/webdriverio/webdriverio/assets/113699426/d94d5bbb-79e1-47ac-8644-836a05ba3de3)
Please let me know how to solve this issue.

**Comment by christian-bromann** [comment]
Can you provide a minimal reproducible example for this please?

**Reply by shanmswa** [reply]
Here is an Example
describe('Image Comparison', async() =>
{
    beforeEach(async () => {
        await browser.url('')
        await browser.pause(3000)
    })
    // save the element in the base line folder
    it('Getting the element of the image', async() => 
    {
        const element = $("")
        await element.waitForDisplayed({})
        await browser.saveElement(element,'Logo',{})
    })

**Reply by christian-bromann** [reply]
This example is not reproducible, I need something I can run on my machine with an application where I can inspect the elements.

**Comment by shanmswa** [comment]
Please find the example
describe('Mocha example with save functionality', () =>{
    beforeEach(async () => {
        await browser.url('https://webdriver.io')
    })
it('should save some screenshots', async () => {
       // Save an element
    await browser.saveElement(
            await $("//a[normalize-space()='Why WebdriverIO?']"),
            'firstWebsiteElement',
            {}))
})

**Reply by christian-bromann** [reply]
Can you provide a git repository for me to check out?
